### DISCUSSION TITLE
is that possible to show the Chinese characters of the rom file names properly?

### DISCUSSION THREAD

**Discussion by bobdos** [eduardofilo/RG350_adam_image#283] [SimpleMenu topics]
is that possible to show the Chinese characters of the rom file names properly in simplemenu?

**Comment by eduardofilo** [comment]
This is something that belongs to the SimpleMenu development. It was discussed [here](https://github.com/eduardofilo/RG350_adam_image/discussions/256#discussioncomment-3078217).

There is a fork of SimpleMenu that seems to be adapted to that language.
https://github.com/lualiliu/simplemenuCN

**Reply by bobdos** [reply]
many thanks for your information.
